#goal definition
bool PERFORM_PRE_GRASP
tue_manipulation/cart_pos goal
tue_manipulation/cart_pos delta

# In case of PERFORM_PRE_GRASP
bool FIRST_JOINT_POS_ONLY # If TRUE, only the first JointTrajectoryPoint
                          # is passed to the JointTrajectoryAction
                          # to be able to recalibrate before the actual
                          # grasp motion
                          # If FALSE, the entire trajectory is executed

# an optional list of obstacles that we have semantic information about
# and that can be touched/pushed/moved in the course of grasping;
string[] allowed_touch_objects
---
#result definition
tue_manipulation/cart_pos result
---
#feedback
tue_manipulation/cart_pos feedback
